lib/philomena_web/controllers/api/json/image_controller.ex

```diff
@@ -27,4 +35,72 @@ defmodule PhilomenaWeb.Api.Json.ImageController do
         render(conn, "show.json", image: image, interactions: interactions)
     end
   end
+
+  def create(conn, %{"image" => image_params} = params) do
+    user = conn.assigns.current_user
+
+    params =
```

Comment:
Why is this necessary? Why can't you just use `image_params`?

Reply:
Yeah. Worked it better so it uses `image_params` and only adds the things from `params` that are strictly required to make the image (metadata, `scraper_url`, `anonymous`).

Reply:
This should be much neater now.